tools/spi-hdlc-adapter/spi-hdlc-adapter.c

```diff
@@ -141,10 +141,6 @@ static int sMode = MODE_PTY;
 static int sMode = MODE_STDIO;
 #endif
 
-static const char *sSpiDevPath     = NULL;
-static const char *sIntGpioDevPath = NULL;
-static const char *sResGpioDevPath = NULL;
```

Comment:
Even though these variable are not used in the code, I suggest we keep them as they are (they would be useful).
For example, in local tests/builds, we used to add these in log messages which was very helpful for debugging, or use gdb to check the value.

Actually I think it  may be helpful to log these as part of the code where the stat/info is being logged. 

Thoughts?

Reply:
I just felt strange when I looked at the these variables. Because no one is using these variables.  For debugging, I think it is better to log these variables to syslog when the code is processing these variables.  Thoughts?

Reply:
I am fine with that.